test/unittests/wasm_engine_test.cpp

```diff
@@ -0,0 +1,69 @@
+#include <gtest/gtest.h>
+#include <test/utils/hex.hpp>
+#include <test/utils/wasm_engine.hpp>
+
+using namespace fizzy;
+using namespace fizzy::test;
+
+TEST(wasm_engine, multi_i32_args_ret_i32)
+{
+    /* wat2wasm
+    (func $test (export "test") (param $a i32) (param $b i32) (param $c i32) (result i32)
+      local.get $a
+      local.get $c
+      i32.sub
+      local.get $b
+      i32.mul
+    )
+    */
+    const auto wasm = from_hex(
+        "0061736d0100000001080160037f7f7f017f03020100070801047465737400000a0c010a00200020026b20016c"
+        "0b");
+
+    for (auto engine_create_fn : {create_fizzy_engine, create_wabt_engine, create_wasm3_engine})
+    {
+        auto engine = engine_create_fn();
+        ASSERT_TRUE(engine->parse(wasm));
+        ASSERT_TRUE(engine->instantiate());
+        auto const func = engine->find_function("test");
```

Comment:
```suggestion
        const auto func = engine->find_function("test");
```
also in three cases below

Reply:
Fixed.